backend/api/v1/endpoints/connections.py

```
@@ -56,13 +56,38 @@ def list_connections(db: Session = Depends(get_db), current_user: User = Depends
     try:
         if not current_user:
             raise HTTPException(status_code=401, detail="Unauthorized")
 
-        users = db.query(User).filter(User.id != current_user.id).all()  # ✅ Use current_user.id
+        # Fetch users who are already friends
+        friends = db.query(Connection.friend_id).filter(Connection.user_id == current_user.id)
+
+        # Fetch users who have been sent a request by the current user
+        sent_requests = db.query(Connection.friend_id).filter(
+            Connection.user_id == current_user.id, 
+            Connection.status == "pending"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)